client/scss/components/_header.scss

```diff
@@ -0,0 +1,289 @@
+//* @define header; weak
+
+.header {
+  padding-top: 0.8rem;
+}
+
+.header__upper-wrap {
```

Comment:
If we're going to be using things like `wrap` - could we maybe document it somewhere explaining that we have:
* .el
* .el-wrap

Or, as I think is actually the case here:
* .el-wrap
* .el

Personally I am a fan of the prior - keeping everything in the `.el` hich seems easier to work out, but the having a `.el-inner` or something similar.